funflow/src/Control/Funflow/Exec/Simple.hs

```diff
@@ -149,7 +149,7 @@ runFlowEx _ cfg store runWrapped confIdent flow input = do
           submitTask po td
           wait chash td
         wait chash td = do
-          KnownTask _ <- awaitTask po chash
+          _ <- awaitTask po chash
```

Comment:
Maybe this, to avoid silent failures:
```suggestion
          awaitTask po chash >>= \case
            KnownTask _ -> pure ()
            _ -> error "[Control.Funflow.Exec.Simple.runFlowEx] Expected KnownTask."
```